modules/installing-using-assisted-installer-oci.adoc

```diff
@@ -0,0 +1,156 @@
+include::snippets/technology-preview.adoc[]
+
+You can access {oci} Resource Manager configurations to provision and configure {oci} resources that meet your organization's needs.
+
+[IMPORTANT]
+====
+The steps for provisioning {oci} resources are provided as an example only. You can also choose to create the required resources through other methods; the scripts are just an example. Installing a cluster with infrastructure that you provide requires knowledge of the cloud provider and the installation process on {product-title}. You can access {oci} Resource Manager configurations to complete these steps, or use the configurations to model your own custom script.
+====
+
+Follow the steps in the quick-start procedure to understand how to use the Assisted Installer to install a {product-title} cluster on OCI. This quick start demonstrates the use of the {oci}'s Manager (CCM) and Oracle’s Container Storage Interface (CSI) objects to link your {product-title} cluster with {oci}'s API.
+
+.Prerequisites
+
+* You reviewed details about the {product-title} xref:../../architecture/architecture-installation.adoc#architecture-installation[installation and update] processes. 
+//* You completed the [Request Access to Red Hat OpenShift on Oracle Cloud Infrastructure in Developer Preview](https://docs.google.com/forms/d/e/1FAIpQLSdQHmw0BUWjQxPKC5_G7lk2_o9Pcnn2ON84Al6s2Ir254gv6Q/viewform) form.
+* If you use a firewall and you plan to use a Telemetry service, you xref:../../installing/install_config/configuring-firewall.adoc#configuring-firewall[configured your firewall], so that {product-title} can access the sites required.
+* You configured an {oci} account to host the cluster. See link:https://docs.oracle.com/en/cloud/get-started/subscriptions-cloud/csgsg/create-users-and-assign-roles.html#GUID-1041AC78-9E43-42CF-B5A5-73F8B8FEB39A[Create Users and Assign Roles]] in the Oracle documentation. 
+* If the cloud identity and access management (IAM) APIs are not accessible in your environment, or if you do not want to store an administrator-level credential secret in the `kube-system` namespace, you can manually create and maintain IAM credentials. See link:https://docs.oracle.com/en-us/iaas/Content/Identity/Concepts/overview.htm[Overview of Identity and Access Management] in the Oracle documentation. 
+* You set up your cloud infrastructure, such as compute, networking, and storage settings, before installing {product-title}. You must ensure that your infrastructure meets the minimum Assisted Installer link:https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform/2022/html-single/assisted_installer_for_openshift_container_platform/index#hardware[hardware requirements]. Before you create your VMs, refer to {op-system-base-full} certified shapes, instance types, to identify the supported {oci} VM shapes. See link:https://catalog.redhat.com/cloud/detail/216977[Cloud instance types] on Red Hat Ecosystem Catalog portal.
```

Comment:
This AI version is obsolete.

Reply:
Hi @apinnick . Does not comment target https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform/2023/html/assisted_installer_for_openshift_container_platform/index#hardware ?

Would you have a link to the latest version of the doc?